res/values/strings.xml

```diff
@@ -177,7 +177,8 @@
     <string name="click_to_see_change_list">Click to see change list</string>
 
 	<string name="changelog_title">Changelog</string>
-	<string name="unranked_features">↓ Unranked features ↓</string>
+	<string name="unranked_features">↓ Unranked features ↓</string>*/
```

Comment:
?

Reply:
i don't remember doing this probably was ide wait